src/modules/RemediationsModal/steps/reviewSystems.js

```diff
@@ -18,18 +18,35 @@ const ReviewSystems = ({ issues, systems, allSystems, registry, ...props }) => {
   const formOptions = useFormApi();
 
   const error = formOptions.getState().errors?.systems;
+  const [bootcError, setBootcError] = useState('');
 
   const rowsLength = useSelector(
     ({ entities }) => (entities?.rows || []).length
   );
   const selected = useSelector(({ entities }) => entities?.selected || []);
+  const bootcIds = useSelector(
+    ({ entities }) =>
+      entities?.rows
+        ?.filter((r) => r.system_profile?.bootc_status?.booted?.image)
+        .map((r) => r.id) || []
+  );
   const loaded = useSelector(({ entities }) => entities?.loaded);
   const allSystemsNamed = useSelector(
     ({ hostReducer: { hosts } }) =>
       hosts?.map((host) => ({ id: host.id, name: host.display_name })) || []
   );
 
   useEffect(() => {
+    const hasBootc = selected.some((s) => bootcIds.includes(s));
+    const isAdvisor = issues?.some((i) => i?.id?.startsWith('advisor'));
+
+    if (hasBootc && !isAdvisor) {
+      setBootcError(
+        'Image mode systems cannot be added to a remediation playbook.'
+      );
+      return input.onChange({});
+    }
+    setBootcError('');
```

Comment:
This doesn't seem right. Won't this make the `bootcError` always defined and evaluate to truthy below in the render?

Reply:
Yea it's an empty string which evaluates to false.